airbyte-migration/README.md

```diff
@@ -2,6 +2,8 @@
 
 This module migrates configs specified in `airbyte-config` to new versions.
 
+WARNING: the file-based migrations are deprecated. Please write a Flyway migration whenever you want to update the database. See [here](../airbyte-db/lib/README.md) for details.
```

Comment:
What happens if we want to migrate one of the json blobs that don't have a db schema today? Would we also write the migration in flyway and within the migration perform the json migration logic?

Reply:
Yes. If a required field needs to be added to the json_blob, a migration is needed to do that.

Today it won't be reflected in the database schema dump. But in issue #5479, we will normalize the json blob, and create separate tables for each config type. Then this kind of schema change will show up in the schema dump.